### ISSUE TITLE
Remove markup between landmarks

### ISSUE THREAD

**Issue by adamwoodnz** [WordPress/Learn#2501]
Currently the site footer sits between the `main` and `footer` landmarks. For accessibility reasons ideally there should be no markup between or wrapping these:

![Image](https://github.com/WordPress/Learn/assets/1017872/e436c07b-4fb1-42e8-af80-f60db8664a91)

The ideal solution would have the tag immediately after `main` be the `footer` with the global footer using a div, to maintain valid html. This requires changes in the global footer block to support changing the `tagName`.
